### DISCUSSION TITLE
MATE Desktop and Nvidia driver compatibility

### DISCUSSION THREAD

**Discussion by numerical2017** [selkies-project/docker-selkies-egl-desktop#12] [General]
Dear Kim,
I saw the latest commit with the switch from MATE desktop to XFCE. I currently use MATE but I know that XFCE is more often adopted for  these applications. Could you please suggest me where I can find more information about the incompatibility you mentioned?
Moreover, before your latest commit, I would have suggested to change the VNC server in order to have full bidirectional clipboard and remote resizing feature with noVNC. Do you think it would be a good improvement? Thank you, best regards, Luca.

**Comment by ehfd** [comment]
The information is available in the following links:
https://ubuntu-mate.community/t/can-anyone-here-upgrade-to-nvidia-driver-510-without-login-problems/25147/4
https://forums.developer.nvidia.com/t/cant-boot-after-upgrading-to-510-driver-on-ubuntu-20-04/202625/18
https://bugs.launchpad.net/ubuntu/+source/xorg-server/+bug/1959995

I also like MATE more, but unfortunately had to switch to Xfce to resolve the driver issue ASAP without monkey patching MATE (although it's not really their faults) and also to prevent future issues arising.
Thanks for reaching out!

**Comment by ehfd** [comment]
> Moreover, before your latest commit, I would have suggested to change the VNC server in order to have full bidirectional clipboard and remote resizing feature with noVNC. Do you think it would be a good improvement? Thank you, best regards, Luca.

I am actually looking for alternative options which has better performance than noVNC, but I am welcomed to review the PR. Please note that however the stack on whole may be substituted later on to a completely different solution.
